policyengine_us/parameters/gov/states/ms/tax/income/credits/non_refundable.yaml

```diff
@@ -0,0 +1,8 @@
+description: Mississippi allows these non-refundable tax credits.
+values:
+  2021-01-01:
```

Comment:
```suggestion
  2021-01-01: {}
  
  2023-01-01:
```

Reply:
> 

Can't pass the test because "Missing 'value' property for gov.states.ms.tax.income.credits.non_refundable[2021-01-01]"

Reply:
use [] instead of {}

Reply:
> use [] instead of {}

I tried. Still can't pass test.

Reply:
we use [] and you need an input for 2023 as well